packages/ui-components/src/Input.tsx

```diff
@@ -6,6 +6,7 @@ import React from 'react';
 import { TextInputArea } from './Shared.styles';
 type Input$Type = 'number' | 'password' | 'text';
 import { isUndefined } from '@polkadot/util';
+import Labelled from './Labelled';
 
 type Props = {
```

Comment:
you can use `interface Props extends SuiInputProps` (in the sui package), so that you don't write the props again.

Then, do `const { customProp1, customProp2, ...otherProps } = this.props`, and `<TextInputArea ...{otherProps}>`. Also, I think TextInputArea belongs to this file, I don't think TextInputArea will be used by anyone else, since they should use this Input directly.

Reply:
From looking at the type definitions on `semantic-ui-react/.../Input.d.ts` it doesn't seem there's a whole lot of overlap to make this a vast improvement, though I made a commit with the changes anyway, in case I'm missing the point. @amaurymartiny